tests/azure_iot/testdata/azure_iot.txt

```diff
@@ -0,0 +1,37 @@
+[!exec:bash] stop
+[!exec:grep] stop
+[!exec:cat] stop
+[!exec:curl] stop
+
+exec docker run  --name azure_iot  dautovri/eden-azure-iot-test
+exec docker cp  azure_iot:/disk  disk/
+exec docker rm -f azure_iot
+
+eden -t 10m pod deploy -p 8028:80 --vnc-display=1 file://disk/Ubuntu-18.04-bionic --disks=file://disk/Ubuntu-18.04-bionic-1 --disks=file://disk/Ubuntu-18.04-bionic-2 --disks=file://disk/Ubuntu-18.04-bionic-3 -n azureiot --memory 3GB -v debug
+stdout 'deploy pod azureiot with ubuntu qcow2 disks request sent'
+
+#detecting
+eden -t 10m pod ps
+cp stdout pod_ps
+grep '^azureiot\s*' pod_ps
+
+# Wait 1 min for loading of apps
+exec sleep 10m
+
+# Wait results
+exec -t 10m bash get.sh azureiot
+
+# Test's config file
+-- eden-config.yml --
+test:
+    controller: adam://{{EdenConfig "adam.ip"}}:{{EdenConfig "adam.port"}}
+    eve:
+      {{EdenConfig "eve.name"}}:
+        onboard-cert: {{EdenConfigPath "eve.cert"}}
+        serial: "{{EdenConfig "eve.serial"}}"
+        model: {{EdenConfig "eve.devmodel"}}
+
+
+-- get.sh --
+address=`grep "^$1"'\s*' pod_ps | cut -f 5`
+until curl $address; do sleep 5; done
```

Comment:
In my experiments, application do not respond on 80 port. As I understand it is expected behavior. 
Can we move our test onto testing logs of our application? I mean, we can try to find something inside `./eden pod logs --fields=app --format=json azure_iot`?

Reply:
Changed on` ./eden pod logs --fields=app --format=json azure_iot ` and grep